com.avaloq.tools.ddk.check.core.test/pom.xml

```diff
@@ -9,8 +9,4 @@
 
   <artifactId>com.avaloq.tools.ddk.check.core.test</artifactId>
   <packaging>eclipse-test-plugin</packaging>
-  <properties>
-    <pmd.ruleset>${pmd.test.ruleset}</pmd.ruleset>
-    <checkstyle.configLocation>${checkstyle.test.configLocation}</checkstyle.configLocation>
-  </properties>

```

Comment:
Why are these properties removed? This is our pattern to distinguish between normal and test settings...

Reply:
We discussed the options (Roman and I) and cam to the conclusion that we actually do 1 check of the entire workspace with the non-test settings, to then scan again with the test settings in each plugin. Failure happens either way. We decided to remove the test settings and just cleanup the PMD to match.

Reply:
Why's that, where can I see it? If it is caused by the aggregate option I activated for PMD I would rather remove this one.

Reply:
We can still remove the aggregate option, but Robbie's change also showed that there is not such a big difference between test config and productive config. So independently form activating or not the aggregate option we can ask a question if we need two different configs. Verify job suggests that tests pass the stricter rules.

Reply:
Turning aggregate off makes no difference here.  I wonder what else changed? As far as I can see, not much in the pom should have caused this.

Reply:
Summary: We agreed to not have different settings for productive and test code because our tests pass the stricter prod rules anyway and we prefer the simpler setup.